src/main/include/Constants.h

```diff
@@ -421,7 +470,7 @@ constexpr double kClimberSetI = 0;
 constexpr double kClimberSetD = 0;
 
 // Maximum arm extension distance
-constexpr double kMaxArmDistance = 20;
+constexpr double kMaxArmDistance = 1000;
```

Comment:
We should find a sane number for this

Reply:
And fix the axis so that it goes back down even when at the max